README.md

```diff
@@ -1,4 +1,5 @@
-[![CircleCI](https://circleci.com/gh/microsoft/CCF/tree/master.svg?style=svg&circle-token=4baa7c8b286e0d8b6c2f320f9f6427e156fa8562)](https://circleci.com/gh/microsoft/CCF/tree/master) [![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20Github%20CI?branchName=master)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=3&branchName=master)
+[![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20Github%20CI?branchName=master)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=3&branchName=master)
+[![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20Github%20CI%20-%20No%20SGX?branchName=master)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=2&branchName=master)
```

Comment:
Can we add labels for these? Or put them in a table?

They produce identical side-by-side badges on the main page, you need to click-through to see that one is SGX and the other non-SGX.

Reply:
I haven't found a way to tweak the badges, so I'll make a table.